packages/components/src/core/Button/index.stories.tsx

```diff
@@ -56,121 +62,193 @@ export const Default = {
-          {text}
-        </RawButton>
-        <RawButton
-          {...props}
-          startIcon={<Icon sdsIcon="download" sdsSize="s" sdsType="button" />}
-          sdsStyle="square"
-          sdsType="secondary"
-        >
-          {text}
-        </RawButton>
-      </div>
-    );
-  },
+const disabledLevel = {
+  display: "contents",
+};
+const stateLevel = {
+  marginBottom: 10,
 };
 
-const minimalPlacementStyles = {
-  display: "grid",
-  gridColumnGap: "24px",
-  gridRowGap: "0px",
-  gridTemplateColumns: "repeat(3, min-content)",
-  gridTemplateRows: "1fr",
+const styleLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 6",
+  marginBottom: 0,
+};
+const typeLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 6",
+  borderStyle: "solid none none none",
+  borderWidth: "2px",
+  justifySelf: "stretch",
+  margin: "20px 0",
+  paddingTop: 10,
+};
+const iconLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 6",
+  borderStyle: "solid none none none",
+  borderWidth: "1px",
+  justifySelf: "stretch",
+  alignSelf: "end",
+  margin: "0 0 5px 0",
+  paddingTop: 10,
+};
+const disabledLabel = {
+  fontWeight: "normal",
+  gridColumn: "1 / 2",
+  alignSelf: "end",
+  marginTop: 0,
+};
+const stateLabel = {
+  fontWeight: "normal",
+  margin: "10px 0",
 };
 
-export const MinimalLivePreview = {
+export const LivePreview = {
   parameters: {
     snapshot: {
       skip: true,
     },
   },
   render: (props: Args): JSX.Element => {
+    // loop through all SDS_STYLES
     return (
-      <div style={minimalPlacementStyles as React.CSSProperties}>
-        <RawButton {...props} sdsStyle="minimal" sdsType="primary">
-          {text}
-        </RawButton>
-
-        <RawButton
-          {...props}
-          startIcon={<Icon sdsIcon="download" sdsSize="s" sdsType="button" />}
-          sdsStyle="minimal"
-          sdsType="primary"
-        >
-          {text}
-        </RawButton>
-        <RawButton {...props} sdsStyle="minimal" sdsType="secondary">
-          {text}
-        </RawButton>
-      </div>
+      <>
+        {SDS_STYLES.map((sdsStyle) => {
+          return <ButtonStyleOption sdsStyle={sdsStyle} key={sdsStyle} />;
+        })}
+      </>
     );
+
+    // loop through all SDS_TYPES + create headers for SDS_STYLES
+    function ButtonStyleOption({ sdsStyle }) {
+      return (
+        <div style={styleLevel}>
+          <h1 style={styleLabel}>
+            Style: <b>{sdsStyle}</b>
+          </h1>
+          {SDS_TYPES.map((type) => {
+            return (
+              <ButtonTypeOption sdsStyle={sdsStyle} type={type} key={type} />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all ICON_OPTIONS + create headers for SDS_TYPES
+    function ButtonTypeOption({ sdsStyle, type }) {
+      return (
+        <div style={displayContents}>
+          <h3 style={typeLabel}>
+            Type: <b>{type}</b>
+          </h3>
+          {type === "primary" ? (
```

Comment:
For the `minimal` style, there is no "secondary" sdsType button with an icon. So this code checks for the type and if it's primary, it loops through the two icon options (`undefined` and a download icon), but if not primary, it manuualy assigns the first icon option from the array of options (`undefined`, line 172)